core/commands.go

```diff
@@ -125,6 +125,14 @@ var (
 		Arity:    -3,
 		KeySpecs: KeySpecs{BeginIndex: 1, Step: 1},
 	}
+	expiretimeCmdMeta = DiceCmdMeta{
```

Comment:
nit: Please add the metadata for new commands at the bottom of the list, this helps us with certain documentation related automation that we run.

Reply:
Moved to bottom of the list.